src/spikeinterface/extractors/shybridextractors.py

```diff
@@ -92,6 +92,15 @@ def write_recording(recording, save_path, initial_sorting_fn, dtype='float32', v
             Type of the saved data. Default float32.
         **write_binary_kwargs: keyword arguments for write_to_binary_dat_format() function
         """
+        try:
+            import hybridizer.io as sbio
+            import hybridizer.probes as sbprb
+            import yaml
+
+            HAVE_SBEX = True
+        except ImportError:
+            HAVE_SBEX = False
```

Comment:
Were these changes supposed to be on a different branch? Not that it matters much

Reply:
I am slowly moving imports inwards. Probably should be in a different branch but I don't think it matters that much.